lekcja3/src/main/java/org/example/Manager.java

```diff
@@ -0,0 +1,18 @@
+package org.example;
+
+public class Manager extends BaseEmployee{
+    private final int baseSalary = 5000;
```

Comment:
zauważ w treści zadania, że "pensja podstawowa pracownika manager **domyślnie** wynosi 5000 zł." Domyślnie - czyli, istnieje możliwość ustawić tam inną pensję. Proponuję zrobić dodatkowy konstruktor w którym będzie się ustawiało wartość salary. To samo dotyczy TicketSellera, tylko tam domyślnie ma być 3000.